lib/compare-file-size.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const promisify = require('bluebird').promisify;
+const glob = promisify(require('glob').glob);
+const getFileSizesPerType = require('./get-filesizes-per-type');
+
+function compareFileSize(sourceFolder, tempFolder) {
```

Comment:
Although I do agree that term `tempFolder` is wrong, I do think this module needs to know about the two folders. We use it here to compare the file sizes of the source and dest folders, but this module can be used on any two folders. If we were to take out this module and release it on its own, the naming `sourceFolder` and `tempFolder`/`destFolder` are wrong, but I don't know a better naming. Thoughts?